observatory_platform/dataset_api.py

```diff
@@ -139,14 +140,14 @@ def to_dict(self) -> Dict:
             dag_id=self.dag_id,
             entity_id=self.entity_id,
             dag_run_id=self.dag_run_id,
-            created=self.created.to_iso8601_string(),
-            modified=self.modified.to_iso8601_string(),
-            data_interval_start=pendulum_to_bq_timestamp(self.data_interval_start),
-            data_interval_end=pendulum_to_bq_timestamp(self.data_interval_end),
-            snapshot_date=pendulum_to_bq_timestamp(self.snapshot_date),
-            partition_date=pendulum_to_bq_timestamp(self.partition_date),
-            changefile_start_date=pendulum_to_bq_timestamp(self.changefile_start_date),
-            changefile_end_date=pendulum_to_bq_timestamp(self.changefile_end_date),
+            created=_dt(self.created),
+            modified=_dt(self.modified),
+            data_interval_start=_dt(self.data_interval_start),
+            data_interval_end=_dt(self.data_interval_end),
+            snapshot_date=_dt(self.snapshot_date),
+            partition_date=_dt(self.partition_date),
+            changefile_start_date=_dt(self.changefile_start_date),
+            changefile_end_date=_dt(self.changefile_end_date),
```

Comment:
I think these changes can also be reverted, as they just needed to be converted into iso8601 strings.